src/encoded/static/components/item-pages/components/tables/ExperimentSetTables.js

```diff
@@ -76,23 +90,29 @@ ExperimentSetsTableTabView.getTabObject = function(props){
     };
 };
 
-export function ExperimentSetsTableTabViewTitle({ totalCount, searchHref }) {
-    const linkText =
-        searchHref && typeof searchHref === 'string' && searchHref.indexOf('/browse/') > -1 ?
-            'Open In Browse View' : 'Open In Search View';
+/**
+ * @todo Eventually maybe add UI controls for selecting columns and other things into here.
+ */
+export function ExperimentSetsTableTabViewTitle(props) {
+    const {
+        totalCount,
+        href: currentSearchHref,
+        externalSearchLinkVisible = false
+    } = props;
+    const linkText = currentSearchHref && typeof currentSearchHref === 'string' && currentSearchHref.indexOf('/browse/') > -1 ?
+        'Open In Browse View' : 'Open In Search View';
     return (
         <h3 className="tab-section-title">
             <span>
                 {typeof totalCount === "number" ? <span className="text-500">{totalCount + " "}</span> : null}
                 {"Experiment Set" + (typeof totalCount === "number" && totalCount !== 1 ? "s" : "")}
             </span>
-            {searchHref ?
-                <a href={searchHref} className="btn btn-primary pull-right" style={{ marginTop: '-10px' }} data-tip="Run embedded search query in Browse/Search View">
+            { externalSearchLinkVisible && currentSearchHref ?
```

Comment:
Figured we can just pass into and handle here without really needing to pass it through EmbeddedItemSearchTable